Docs/reference/content/what_is_new.md

```diff
@@ -8,6 +8,12 @@ title = "What's New"
   pre = "<i class='fa fa-star'></i>"
 +++
 
+## What's New in 2.9.0
+
+Some of the changes in 2.9.0 include:
+
+* Retryable writes are enabled by default
```

Comment:
Just noting that if a user has custom retry logic and uses the old default of false, then after upgrading to a version that uses the default of true, their app could end up retrying twice. Probably not worth classing as a breaking change though...